.expeditor/templates/verify_shared_pipeline.yml

```diff
@@ -4,3 +4,10 @@
     expeditor:
       executor:
         docker:
+
+  - label: "[@@plan@@] :linux: :habicat: Check for bad patterns"
```

Comment:
Should `:linux:` be removed if this is no longer in the Linux pipeline config?

Reply:
Maybe? 🤦‍♂️   Fixed!